src/terraform-mcp-server/uv.lock

```diff
@@ -179,7 +179,7 @@ wheels = [
 
 [[package]]
 name = "awslabs-terraform-mcp-server"
-version = "0.0.5"
+version = "0.0.6"
```

Comment:
update the `pyproject.toml` and the `awslabs/terraform_mcp_server/__init__.py:__version__`